NOTICE

```diff
@@ -0,0 +1,22 @@
+Envoy
+Copyright (c) 2016-2017 Lyft Inc.  All rights reserved.
+
+Licensed under Apache License 2.0.  See LICENSE for terms.
+
+Zipkin (Apache V2)
```

Comment:
None of the libraries listed here are part of this repository, so this doesn't belong here.